easybuild/easyconfigs/r/R/R-3.3.0-foss-2016a.eb

```diff
@@ -0,0 +1,455 @@
+name = 'R'
+version = '3.3.0'
+
+homepage = 'http://www.r-project.org/'
+description = """R is a free software environment for statistical computing and graphics."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://cran.us.r-project.org/src/base/R-%(version_major)s']
+
+preconfigopts = 'BLAS_LIBS="$LIBBLAS" LAPACK_LIBS="$LIBLAPACK"'
+configopts = "--with-lapack --with-blas --with-pic --enable-threads --with-x=no --enable-R-shlib --with-tcltk "
+configopts += '--with-tcl-config="$EBROOTTCL"/lib/tclConfig.sh --with-tk-config="$EBROOTTK"/lib/tkConfig.sh '
```

Comment:
this is new, we don't seem to have this in other R easyconfigs

why did you (have to) add it?

is it related to https://github.com/hpcugent/easybuild-easyconfigs/issues/1863 at all?